dapps/assets/src/components/contacts/Contacts.ts

```diff
@@ -121,17 +121,21 @@ export default class ContactsComponent extends mixins(EvanComponent) {
   async onUpdateContact(updatedContact: Contact): Promise<void> {
     await updateContactDispatcher.start(this.getRuntime(), updatedContact);
     // Optimistic updating of the contact
-    this.contacts = this.contacts.map((item: Contact) => {
-      const contact = item;
-      if (item.address === updatedContact.address) {
-        contact.alias = updatedContact.alias;
-      }
-      return contact;
-    });
+    this.contacts = this.contacts
+      .map((item: Contact) => {
+        const contact = item;
+        if (item.address === updatedContact.address) {
+          contact.alias = updatedContact.alias;
+        }
+        return contact;
+      });
   }
 
   async fetchContacts(): Promise<Contact[]> {
-    return this.contactService.getContacts();
+    const runtime = this.getRuntime();
+    const contacts = await this.contactService.getContacts();
+    return contacts.filter((item: Contact) => item.address !== runtime.activeAccount
+      && item.address !== runtime.activeIdentity);
```

Comment:
Is this to filter out the own account? If so, it should be handled in the ContactService.getContacts() already. If it doesnt work, it needs to be fixed there (line 51) instead of fixing it in the contacts.ts :)

Reply:
I assume it's a problem stemming from identity based profiles.

Reply:
Should be fixed now ;)

Reply:
sexy 😍